<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>countdown</class>
 <widget class="QWidget" name="countdown">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>-Countdown</string>
  </property>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>281</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>Take break,every    minutes.</string>
   </property>
  </widget>
  <widget class="QSlider" name="slider_time">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>6</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
   <property name="sliderPosition">
    <number>1</number>
   </property>
   <property name="tracking">
    <bool>true</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
   <property name="invertedControls">
    <bool>false</bool>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksBelow</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_time">
   <property name="geometry">
    <rect>
     <x>152</x>
     <y>49</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>231</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string> Show notification message</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Play sound:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Custom notification message:</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>179</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::CustomContextMenu</enum>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QDial" name="dial_time">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>50</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>9</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="tracking">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
   <property name="invertedControls">
    <bool>false</bool>
   </property>
   <property name="wrapping">
    <bool>false</bool>
   </property>
   <property name="notchesVisible">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_countTime">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>320</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QComboBox" name="playsound">
   <property name="geometry">
    <rect>
     <x>118</x>
     <y>218</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentText">
    <string>(none)</string>
   </property>
   <item>
    <property name="text">
     <string>(none)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>-Default-</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Notify</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>220</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/src/src/play-icon.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
   <property name="intValue" stdset="0">
    <number>7</number>
   </property>
  </widget>
  <widget class="QPushButton" name="quit">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>320</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>quit</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tab_countd">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>351</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>381</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="documentMode">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="tab_1">
    <attribute name="title">
     <string>倒计时</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>...</string>
    </attribute>
   </widget>
  </widget>
  <zorder>tab_countd</zorder>
  <zorder>label_1</zorder>
  <zorder>slider_time</zorder>
  <zorder>label_time</zorder>
  <zorder>checkBox_1</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>toolButton</zorder>
  <zorder>dial_time</zorder>
  <zorder>pushButton_countTime</zorder>
  <zorder>playsound</zorder>
  <zorder>pushButton</zorder>
  <zorder>lcdNumber</zorder>
  <zorder>quit</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
